nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -95,5 +109,8 @@ workflow {
  * Send completion email
  */
 workflow.onComplete {
-    send_email(summary)
+    def multiqc_report = []
```

Comment:
TBC I guess? Maybe worth a `TODO`?

Reply:
Yep @ggabernet said she was working on this 👍